packages/react/src/ActionList/ActionList.test.tsx

```diff
@@ -378,4 +379,45 @@ describe('ActionList', () => {
     const heading = getByText('Group Heading')
     expect(list).toHaveAttribute('aria-label', heading.textContent)
   })
+
+  it('should render ActionList.Item as button when feature flag is enabled', async () => {
+    const {container} = HTMLRender(
+      <FeatureFlags flags={{primer_action_list_item_as_button: true}}>
```

Comment:
There was [a recent PR](https://github.com/primer/react/pull/4615) by @joshblack to address this! I just needed to add `primer_react` instead of just `primer`. Should be good now!